### PR TITLE
feat(connector): make official connectors scope configurable

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Make official connectors' (non-standard social connectors) scope configurable.
Previously, we only supported fixed scopes, which are not flexible.
TODO: Non-empty scope for the Apple connector should have a different response handler than the current design, will do this later.

## Testing

Tested locally.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/connectors/connector-alipay-web/README.md
Comment (gao-sun) [inline packages/connectors/connector-alipay-web/README.md:1]: do you have an issue for the apple connector update?

Comment (darcyYe) [reply packages/connectors/connector-alipay-web/README.md:1]: Issue added LOG-6040.

Comment (gao-sun) [reply packages/connectors/connector-alipay-web/README.md:1]: actually we do have an issue LOG-3640 for this

**Review by wangsijie — APPROVED** [review]

File: packages/connectors/connector-alipay-web/README.md
Comment (darcyYe) [reply packages/connectors/connector-alipay-web/README.md:1]: Marked LOG-6040 as duplicated.

**Pull request merged**